pkg/kubelet/cm/dra/plugin/client_test.go

```diff
@@ -148,7 +149,8 @@ func TestGRPCConnIsReused(t *testing.T) {
 					},
 				},
 			}
-			client.NodePrepareResources(context.TODO(), req)
+			_, err = client.NodePrepareResources(context.TODO(), req)
+			assert.NoError(t, err)
```

Comment:
> Shouldn't the test fail "loudly" if it gets that error?

Ah, error checking was missing. Never mind.

/lgtm
/assign @klueska 

For approval.

No need to backport because the test isn't flaking in previous releases.